web/content/en/docs/Benchmarks/plant_resource_benchmarks.md

```diff
@@ -0,0 +1,214 @@
+Modules of the other compartments are defined as for the reference plant  (<a href="/docs/benchmarks/plant_resource_benchmarks/#Figure_2">Figure 2a</a>). 
+This means, for example, to test the below-ground module ‘FixedSalinity’, above-ground interaction is disabled (SimpleTest) and mortality is mechanistic.
+
+The general structure of each benchmark is a 2-plant setup, without recruitment (<a href="/docs/benchmarks/plant_resource_benchmarks/#Figure_2">Figure 2b</a>).
+The plants are placed centered in a 22x22 m model domain with a fixed distance of 2 m. 
+The initialization of those plants is based on the geometry of the reference plant. 
+The initial population is designed in a way that the respective geometry triggers a potentially critical situation of the concept. 
+In some below-ground modules, this is for example the case, when the root systems of the plants overlap.
+Thus, we defined an initial population for each compartment. 
+Each initial population is stored in ’Benchmarks/ModuleTests/<plant_module>/\<compartment>/’.
+
+If land- and seaward boundary conditions need to be defined in a setup, salinity is set to 25 ppt at the landward boundary and to 35 ppt at the seaward boundary. 
+Random seed of all setups is 643879.
+
+
+Time step length is 1e6 seconds (~12 days) and simulation time is 5e8 secs (~ 15.8 years, i.e. 500 time steps).
+Output is written after the first and last time step. 
+The first output is used for automatic testing.
+
+<figure class="alert">
+     <img id="Figure_2" src="/pictures/benchmarks/basic_setup.jpg">
+     <figcaption>
+        <i><br><strong>Figure 2:</strong> (a) Overview below-ground interaction setups. (b) Schematic representation of benchmark setup (based on Bathmann et al. 2020).</i>
+     </figcaption>
+</figure>
+
+The following files must be provided for each benchmark
+- pyMANGA project file
+- Results csv file (`<output_times> [2e6, 5e8] </output_times>`) of type `OneTimestepOneFile`
+- All files required to run the setup, e.g. OGS project file (if applicable)
+ 
+The files are stored in 'Benchmarks/ModuleBenchmarks/<plant_module>/\<compartment>/' with the name of the module tested.
+
+## Overview of existing modules and benchmarks
+
+Here is an overview of which modules and benchmarks are available. Modules marked in blue have already been benchmarked. Click on the button to go to the appropriate folder.
+
+### Modules tested with SimpleBettina
+
+SimpleBettina works with all resource and mortality modules and has therefore the most comprehensive collection of benchmarks.
+
+<div class="container">
+    <div class="border rounded p-3 mt-1">
+        <h3>
+            <span class="text-primary">Aboveground</span>
+        </h3>
+        <hr>
+        <a href="https://github.com/pymanga/pyMANGA/tree/master/Benchmarks/ModuleBenchmarks/PlantModules/Bettina/Aboveground" target=”_blank”><span class="tag tag-inactive">NULL</span></a>
```

Comment:
Can you rename all "NULL" Benchmarks to "Default". This is the new/final name for SimpleTest modules.